wgsl/index.bs

```diff
@@ -5776,89 +5776,92 @@ The reason is that only functions without a return type can have such a [=behavi
     <td class="nowrap">{ }
     <td>
     <td class="nowrap">{Next}
-  <tr>
+  <tr algorithm="braced statement behaviour">
```

Comment:
Nit: I used the "behavior" spelling throughout instead of "behaviour", it should probably be kept consistent.

Reply:
Before your PR, there were 13 "behaviour" instances and 1 "behavior" instance. :-)

Reply:
Taking your suggestion (for these new instances)